foreman/resource_foreman_katello_repositories.go

```diff
@@ -346,7 +349,14 @@ func buildForemanKatelloRepository(d *schema.ResourceData) *api.ForemanKatelloRe
 	repo.DebComponents = d.Get("deb_components").(string)
 	repo.DebArchitectures = d.Get("deb_architectures").(string)
 	repo.IgnoreGlobalProxy = d.Get("ignore_global_proxy").(bool)
-	repo.IgnorableContent = d.Get("ignorable_content").(string)
+    if ignorableContent, ok := d.GetOk("ignorable_content"); ok {
+        casted := ignorableContent.([]interface{})
+        var ignorableContentList []string
+        for _, item := range casted {
+            ignorableContentList = append(ignorableContentList, item.(string))
+        }
+	    repo.IgnorableContent = ignorableContentList
```

Comment:
Why don't you directly assert to `[]string` here? Suggestion also adds `ok` to check, else it may panic.

Like this:
```suggestion
    if ignorableContent, ok := d.GetOk("ignorable_content"); ok {
        ignorableContentList, ok := ignorableContent.([]string)
        if ok {
            repo.IgnorableContent = ignorableContentList
        }
    
```